randovania/games/dread/json_data/header.txt

```diff
@@ -111,31 +111,222 @@ Templates
+                      Combat (Intermediate) and Normal Damage ≥ 325
+                  All of the following:
+                      # Two bonks per Screw Attack
+                      Combat (Advanced) and Normal Damage ≥ 175
+                  All of the following:
+                      # Counter to restore health, damage required to survive one hit
+                      Combat (Expert) and Normal Damage ≥ 55
+
 * Fight Gold Robot:
       Any of the following:
-          Lay Power Bomb
+          Knowledge (Beginner) and Lay Power Bomb
           All of the following:
               Any of the following:
                   # Methods to dodge attacks
                   Flash Shift or Combat (Beginner)
               Any of the following:
                   # Methods of fighting
-                  Shoot Missile or Shoot Plasma Beam
-                  Combat (Beginner) and Shoot Charge Beam
+                  Shoot Plasma Beam
+                  Combat (Intermediate) and Shoot Charge Beam
+                  All of the following:
+                      # 54 Missiles are required, 1.5x leeway for trickless
+                      Shoot Missile
+                      Any of the following:
+                          Missiles ≥ 81
+                          Missiles ≥ 54 and Combat (Beginner)
+                  All of the following:
+                      # 34 Super Missiles are required, 1.5x leeway for trickless
+                      Shoot Super Missile
+                      Any of the following:
+                          Missiles ≥ 51
+                          Missiles ≥ 34 and Combat (Beginner)
           All of the following:
               Screw Attack
               Any of the following:
                   All of the following:
                       # One bonk per Screw Attack
-                      Combat (Intermediate) and Normal Damage ≥ 650
+                      Combat (Intermediate) and Normal Damage ≥ 825
                   All of the following:
                       # Two bonks per Screw Attack
-                      Combat (Advanced) and Normal Damage ≥ 325
+                      Combat (Advanced) and Normal Damage ≥ 425
+                  All of the following:
+                      # Counter to restore health, damage required to survive one hit
+                      Combat (Expert) and Normal Damage ≥ 82
 
-* Open Charge Door:
+* Fight Twin Robots:
       Any of the following:
-          Shoot Charge Beam
           Knowledge (Beginner) and Lay Power Bomb
+          All of the following:
+              Any of the following:
+                  # Methods of dodging
+                  Flash Shift or Combat (Beginner)
+              Any of the following:
+                  # Methods of fighting
+                  Shoot Plasma Beam
+                  Combat (Intermediate) and Shoot Charge Beam
+                  All of the following:
+                      # 101 Missiles are required, 1.5x leeway for trickless
```

Comment:
ammo counts for twins could also account for drops from killing one of them, with a combat trick to accompany ofc

Reply:
not sure. this one isn't a huge deal